### PR TITLE
Update Artist Compymono Infomation and his game information

### PR DESCRIPTION
**Pull request by PianCat**

### CODE REVIEW

**Review by ShellWen — COMMENTED** [review]
waiting for confirmation from Compymono personally

File: games/Beek_Familiar_Spirit.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ description-format: markdown
 authors:
   - name: Studio Klondike
     role: [ producer, screenwriter, programmer, artist ]
+  - name: Compymono
+    role: [ programmer, artist ]
```
Comment (ShellWen) [inline games/Beek_Familiar_Spirit.yaml:16]: add "translator"

**Review by ShellWen — CHANGES_REQUESTED** [review]
add "translator" to Beek Familiar Spirit

File: games/Beek_Familiar_Spirit.yaml (hunk 1 above)
Comment (PianCat) [reply games/Beek_Familiar_Spirit.yaml]: @ShellWen Solved

**Pull request merged**